sympy/ntheory/factor_.py

```diff
@@ -2040,3 +2048,83 @@ def eval(cls, n):
                 raise ValueError("n must be a positive integer")
             else:
                 return sum(factorint(n).values())
+
+
+def mersenne_prime_exponent(nth):
+    """
```

Comment:
Start Returns right after """ and leave blank line (but no space) before Examples

Reply:
Done